pkg/controller/iampolicy/iampolicy_controller.go

```diff
@@ -135,6 +148,7 @@ type ReconcileIamPolicy struct {
 // +kubebuilder:rbac:groups=apps,resources=deployments/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=policy.open-cluster-management.io,resources=policies,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=policy.open-cluster-management.io,resources=policies/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=user.openshift.io,resources=groups,verbs=get
```

Comment:
Just FYI: I'm not positive, but the role deployed with ACM might not come from these kubebuilder directives. I think it could be in the helm chart (so an update *might* be needed there)

Reply:
I think so too! I just added it here for consistency.